--- a/cfg80211.c
+++ b/cfg80211.c
@@ -6447,7 +6447,7 @@
 	s32 err = 0;
 
 	cfg->scan_request = NULL;
-	cfg->pwr_save = true;
+	cfg->pwr_save = false;
 	cfg->dongle_up = false;		/* dongle is not up yet */
 	err = brcmf_init_priv_mem(cfg);
 	if (err)
@@ -7371,7 +7371,7 @@
 
 	brcmf_dongle_scantime(ifp);
 
-	power_mode = cfg->pwr_save ? PM_FAST : PM_OFF;
+	power_mode = PM_OFF; /* pwnagotchi: force power-save off, ~4x fewer SDIO crashes */
 	err = brcmf_fil_cmd_int_set(ifp, BRCMF_C_SET_PM, power_mode);
 	if (err)
 		goto default_conf_out;
